### PR TITLE
Javascript-objects

### PR DESCRIPTION
**Pull request by kkmanuu**
Added index.html
Added style.css
I used Gitflow in branching model and workflow for version control in software development projects 
I used Eslint linter tool for JavaScript to help enforce consistent coding practices and identify potential issues in code.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,104 +1,56 @@
...
-}));
-
 class Books {
-  constructor(title, author, id) {
+  constructor(title, author) {
     this.title = title;
     this.author = author;
-    this.id = id;
   }
 }
```
Comment (roshan-bajgain) [inline script.js]: - [ ] According to the project requirement you should create a collection that keeps a list of books. For example: you can use array of object for that and Display all books saved in the collection in the top part of the page.
- [ ] [Optional] In the upcoming project you will use class so I think for now you can use plain javascript function.
### Requirment
his

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kkmanuu and @Log-benjamin, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] I would suggest you please add second author to you README file as it makes your README file professional.
- [ ] Please make sure while writing your commit message it should be in the present form. Good commit message:
`Add file not Added file`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (kkmanuu) [reply script.js]: @roshan-bajgain Done Making changes

**Review by divinecharlotte — APPROVED** [review]
# Hello @kkmanuu      😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Good implementation of functionalities👍 
- Descriptive readme👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**